lib/methods/start.js

```diff
@@ -18,19 +19,30 @@ module.exports = async function start() {
     const { devices, modbus } = settings.get();
     const options = {
       baudRate: modbus.baud_rate,
+      parity: modbus.parity ? modbus.parity : 'none',
```

Comment:
I feel like defaults are handled differently elsewhere. Also, couldn't this be shorter with ```modbus.parity || 'none'```? I don't remember how that handled 0.

Reply:
Nevermind this, would go wrong with handling 0.